public/app/plugins/datasource/graphite/configuration/LokiLogsMappings.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { ChangeEvent, useState } from 'react';
+import { Button, Icon, InfoBox, InlineFormLabel, Input } from '@grafana/ui';
+
+type Props = {
+  mappings: string[];
+  onChange: (mappings: string[]) => void;
+};
+
+export const LokiLogsMappings = (props: Props): JSX.Element => {
+  const [mappings, setMappings] = useState(props.mappings || []);
+
+  return (
+    <div>
+      <h3 className="page-heading">Loki labels mappings</h3>
+      <InfoBox title="How to map Graphite metrics to Loki labels?">
```

Comment:
I think that this InfoBox is too dominant. I am wondering, if adding a button that would show/hide these information wouldn't make sense. I would also check with UX team cause as far as I can remember, I didn't see this pattern in any data source. 

Information in it are super valuable and helped me a lot in understanding how to set this up. So I think it 100% makes sense to have it here. But I think it should be less dominant.

Reply:
True that! It's way too dominant 🙈 We do it some other data sources, though like this bad boy:

<img width="997" alt="Screenshot 2021-04-28 at 09 05 08" src="https://user-images.githubusercontent.com/745532/116361259-12df7480-a801-11eb-9a3d-4bfe219f73ad.png">

But maybe it'd be better to use something we have in Influx?

![help](https://user-images.githubusercontent.com/745532/116361317-212d9080-a801-11eb-8650-5c71c52e16b5.gif)

Reply:
I've add "more button" a fixed spacing between inputs:

![more](https://user-images.githubusercontent.com/745532/116512508-71bbf100-a8c8-11eb-8816-191fd01a14c6.gif)

@ivanahuckova, @jessover9000 - what do you think?

Reply:
I think the info box is not as bad as the other one you showed as an example because it's a bit more compact. (how we show help is still a bit of a general UX issue within Grafana, some of these help boxes are very big.) 

Just for my understanding, is this "How to map Graphite metrics to Loki labels" box permanently going to be visible? I'm wondering whether it would make more sense to only have the title of the info box show in a kind of accordion way, and then on expand you get the entire text and information, instead of having half the info box show first and the rest hidden behind "more"?  🤔 Also I'm wondering whether it should be collapsed by default or expanded by default – maybe have it expanded by default with an "X" / "Hide this information" option and then it just gets compressed to an accordion with title only?

Reply:
I've added an option to dismiss the box (it will stay dismissed after the refresh). It can be brought back by clicking on "Learn more" (felt a bit more natural than accordion but can change it too). By default the info box is visible:

![mappings](https://user-images.githubusercontent.com/745532/116555856-2243e800-a8fd-11eb-8f43-032cd3036c2c.gif)

Reply:
That works nicely! I would actually change the "Learn more" from a button just to a text link, because the button makes it just as important as "Add label mapping" which I think it's not (only additional information that's not needed for everyone after all). I would probably change the link text to "Learn how label mappings work" to make it more concrete. :)

Reply:
Good point! I'll change it 👍

Reply:
Updated version:

<img width="587" alt="Screenshot 2021-04-30 at 11 12 02" src="https://user-images.githubusercontent.com/745532/116674436-eebc9880-a9a4-11eb-93df-15e2fc3f34b0.png">